atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/OldestTargetedSweepTrackedTimestampTest.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.sweep.queue;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.google.common.collect.ImmutableList;
+import com.palantir.atlasdb.keyvalue.api.KeyValueService;
+import com.palantir.atlasdb.keyvalue.impl.InMemoryKeyValueService;
+import com.palantir.atlasdb.table.description.TableMetadata;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.function.LongSupplier;
+import org.jmock.lib.concurrent.DeterministicScheduler;
+import org.junit.Test;
+
+public class OldestTargetedSweepTrackedTimestampTest {
+    private final InMemoryKeyValueService kvs = new InMemoryKeyValueService(true);
+    private final LongSupplier timestampSupplier = mock(LongSupplier.class);
+    private final AtomicBoolean schedulerShutDown = new AtomicBoolean(false);
+    private final DeterministicScheduler deterministicScheduler = new DeterministicScheduler() {
+        @Override
+        public List<Runnable> shutdownNow() {
+            schedulerShutDown.set(true);
+            return ImmutableList.of();
```

Comment:
It seems a bit suspicious that this doesn't call super.shutdownNow() - if intended is probably worth a comment at least

Reply:
Not suspicious at all, the only reason this exists is because super throws an `UnsupportedOperationException` 🗡️

Reply:
I see. It is still suspicious, just not actually wrong 😅  would still suggest the comment